From d4a4e391e63025a655813019bd391e16eff3cd8f Mon Sep 17 00:00:00 2001
From: Etienne JUVIGNY <ti3nou@gmail.com>
Date: Thu, 22 Dec 2022 16:15:28 +0100
Subject: Remove tests, unhappy after the reverts


diff --git a/PKGBUILD b/PKGBUILD
index c403c10730da7ad32ddd3853f06ac4ab4b3b4f92..fafba76e132451f9c8bda57f7cdc91cac39cf833 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -125,37 +125,6 @@ build() {
     DESTDIR="${srcdir}"/locales install-locale-files
 }
 
-# Credits for _skip_test() and check() @allanmcrae
-# https://github.com/allanmcrae/toolchain/blob/f18604d70c5933c31b51a320978711e4e6791cf1/glibc/PKGBUILD
-_skip_test() {
-  test=${1}
-  file=${2}
-  sed -i "/\b${test} /d" "${srcdir}/glibc/${file}"
-}
-
-check() (
-  cd glibc-build
-
-  # adjust/remove buildflags that cause false-positive testsuite failures
-  sed -i 's/-Werror=format-security/-Wformat-security/' config.make   # failure to build testsuite
-  sed -i '/CFLAGS/s/-fno-plt//' config.make                           # 27 failures
-  sed -i '/CFLAGS/s/-fexceptions//' config.make                       # 1 failure
-
-  # The following tests fail due to restrictions in the Arch build system
-  # The correct fix is to add the following to the systemd-nspawn call:
-  # --system-call-filter="@clock @memlock @pkey"
-  _skip_test test-errno-linux        sysdeps/unix/sysv/linux/Makefile
-  _skip_test tst-mlock2              sysdeps/unix/sysv/linux/Makefile
-  _skip_test tst-ntp_gettime         sysdeps/unix/sysv/linux/Makefile
-  _skip_test tst-ntp_gettimex        sysdeps/unix/sysv/linux/Makefile
-  _skip_test tst-pkey                sysdeps/unix/sysv/linux/Makefile
-  _skip_test tst-process_mrelease    sysdeps/unix/sysv/linux/Makefile
-  _skip_test tst-shstk-legacy-1g     sysdeps/x86_64/Makefile
-  _skip_test tst-adjtime             time/Makefile
-
-  make -O check
-)
-
 package_glibc() {
   pkgdesc='GNU C Library'
   depends=('linux-api-headers>=4.10' tzdata filesystem)
