From 5d065a741c8c1005d43090255889240ec031d11f Mon Sep 17 00:00:00 2001
From: Etienne JUVIGNY <ti3nou@gmail.com>
Date: Thu, 22 Dec 2022 16:10:44 +0100
Subject: Add rogue company reverts patch


diff --git a/PKGBUILD b/PKGBUILD
index a783164..3a8bdc5 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -22,6 +22,7 @@ source=(git+https://sourceware.org/git/glibc.git#commit=${_commit}
         reenable_DT_HASH.patch
         fix-malloc-p1.patch
         fix-malloc-p2.patch
+        rogue_company_reverts.patch
 )
 validpgpkeys=(7273542B39962DF7B299931416792B4EA25340F8 # Carlos O'Donell
               BC7C7372637EC10C57D7AA6579C43DFBF1CF2187) # Siddhesh Poyarekar
@@ -31,7 +32,8 @@ b2sums=('SKIP'
         '214e995e84b342fe7b2a7704ce011b7c7fc74c2971f98eeb3b4e677b99c860addc0a7d91b8dc0f0b8be7537782ee331999e02ba48f4ccc1c331b60f27d715678'
         '35e03ed912e1b0cd23783ab83ce919412885c141344905b8b67bbad4a86c48cf3e893806060e48d5737514ff80cea0b58b0e1f15707c32224579c416dcd810c0'
         '28c983bcebc0eeeb37a60756ccee50d587a99d5e2100430d5c0ee51a19d9b2176a4013574a7d72b5857302fbb60d371bbf0b3cdb4fc700a1dbe3aae4a42b04b9'
-        'c3e94f5b0999878ff472e32f49dc13c20eb9db68c633017cb7824617eb824cf6cff7ea53b92962926e0ee84fd39736616298dcb926356625dd124f3754e79932')
+        'c3e94f5b0999878ff472e32f49dc13c20eb9db68c633017cb7824617eb824cf6cff7ea53b92962926e0ee84fd39736616298dcb926356625dd124f3754e79932'
+        'SKIP')
 
 prepare() {
   mkdir -p glibc-build lib32-glibc-build
@@ -43,6 +45,9 @@ prepare() {

   patch -Np1 -i "${srcdir}"/fix-malloc-p1.patch
   patch -Np1 -i "${srcdir}"/fix-malloc-p2.patch
+
+  # Reverting 7a5db2e82fbb6c3a6e3fdae02b7166c5d0e8c7a8, 8208be389bce84be0e1c35a3daa0c3467418f921, 6bf789d69e6be48419094ca98f064e00297a27d5, b89d5de2508215ef3131db7bed76ac50b3f4c205, 86f0179bc003ffc34ffaa8d528a7a90153ac06c6 for Rogue Company to work again
+  patch -Np1 -i "${srcdir}"/rogue_company_reverts.patch
 }
 
 build() {
